### DISCUSSION TITLE
Coral Dev board

### DISCUSSION THREAD

**Discussion by Wojo1122** [blakeblackshear/frigate#3019] [Ask A Question]
So just to make sure, nobody ever got the big dev board working after the update, right?

**Comment by jlpoolen** [comment]
I have the Coral Dev Board -- acquired in April, 2019.  Has anyone installed Frigate on that unit?  Or, has anyone configure the unit to receive/process/send results to another machine?

**Reply by zac9** [reply]
I just started last week. I installed docker-ce on a dev board as per link below, all seams working for now. I have only tested a few cams and have not mounted anything yet. 
 [Namburger commented on Jan 7, 2020](https://github.com/f0cal/google-coral/issues/32#issuecomment-571629174)

**Reply by jlpoolen** [reply]
> I just started last week. I installed docker-ce on aa dev board as per link below, all seams working for now. I have only tested a few cams and have not mounted anything yet. Namburger commented on Jan 7, 2020 [https://github.com/f0cal/google-coral/issues/32#issuecomment-571629174](url)

Link returns 404

**Reply by jlpoolen** [reply]
I found the F0cal [zero instead of capital "O"] project "https://github.com/f0cal /
[google-coral](https://github.com/f0cal/google-coral)" - "Community (read: "unofficial") gathering point for Google Coral dev board and dongle knowledge."

Here is the correct link:  https://github.com/f0cal/google-coral/issues/32

**Reply by zac9** [reply]
sorry fixed link
